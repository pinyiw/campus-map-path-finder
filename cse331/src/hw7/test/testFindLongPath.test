# load graph
LoadGraph g marvel.tsv

# find long path
FindPath g DOC_SAMSON/DR._LEONA LOBO