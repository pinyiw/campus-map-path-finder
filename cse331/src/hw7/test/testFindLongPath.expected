# load graph
loaded graph g

# find long path
path from DOC SAMSON/DR. LEONA to LOBO:
DOC SAMSON/DR. LEONA to HULK/DR. ROBERT BRUC with weight 0.009
HULK/DR. ROBERT BRUC to CAPTAIN AMERICA with weight 0.009
CAPTAIN AMERICA to IRON MAN/TONY STARK  with weight 0.002
IRON MAN/TONY STARK  to LOBO with weight 0.200
total cost: 0.220